### ISSUE TITLE
No animation for the not dragged item

### ISSUE THREAD

**Issue by GreenMonkeyBoy** [kutlugsahin/ngx-smooth-dnd#10]
### Versions

```
* @angular/core: 6.0.2
* ngx-smooth-dnd: 0.2.0
* smooth-dnd: 0.5.4
```

### Observed behavior

No animation for the not dragged item, and the new position of dropped item is sometimes totally wrong.

### Repro steps

Stackblitz: https://stackblitz.com/edit/angular-gitter-hlbdjt

### Mention any other details that might be useful

I tried to import BrowserAnimationModule and install @angular/animations because I know they are requires for some libraries like angular/material, but it didn't fix the problem.

Could it be related to my [last issue](https://github.com/kutlugsahin/ngx-smooth-dnd/issues/9) or maybe I'm just missing something?

**Comment by jeffsapia** [comment]
Downgrade your version to 0.1.9. For some reason angular 6 doesn't want to work with this. Still an amazing library by the way.

**Comment by GreenMonkeyBoy** [comment]
You are right, it works fine downgrading to 0.1.9 : https://stackblitz.com/edit/angular-gitter-ywf8kh. Thank you.

**Comment by kutlugsahin** [comment]
Hey guys sorry for the trouble. Somehow v 0.2.0 has problems with angular 6 I will fix it asap and make sure it uses the latest version of smooth-dnd. Cheers!
